smolog.cpp

```diff
@@ -56,6 +56,23 @@ namespace smolog {
 		remove_sink(s.get());
 	}
 
+	void logger::set_name(const char* name)
+	{
+		if (_internal)
+		{
+			_internal->name = name;
```

Comment:
Please change brace style to match existing.

Reply:
Done!

Reply:
Please change it some more :)